packages/cli/examples/helpers.ts

```diff
@@ -1,101 +1,100 @@
 /* eslint-disable @typescript-eslint/camelcase */
-import {
-  logs
-} from "@cosmwasm/sdk";
+import { FeeTable } from "@cosmwasm/sdk/types/signingcosmwasmclient";
+import axios from "axios";
+import * as fs from "fs";
```

Comment:
Both `axios` and `fs` are no runtime dependencies of `@cosmwasm/cli`, so they just work by luck. If we want to have them, I'd rather imports them by default and add to dependencies.

We already have

```
  let init = `
    import * as http from 'http';
    import * as https from 'https';
  `;
```

which I guess can be replaced with `axios`, since axios API can easily be used in browsers as well. Let me take care of this in a separate PR

Reply:
Ditto on rebase and update